src/main/java/frc/robot/commands/autonomousCommands/AutonomousCommandGroup.java

```diff
@@ -22,9 +23,10 @@
 public class AutonomousCommandGroup extends SequentialCommandGroup {
 
     public AutonomousCommandGroup(Drive drive, ShooterSpeedController shooterSpeedController, HopperController hopperController, 
-                                    IndexerController indexerController, Limelight limelight, double distance, double maxSpeed){
+                                    IndexerController indexerController, Limelight limelight, double distance, double maxSpeed, Lights lights){
         addCommands(
-            new AimAndDriveBackCommand(distance, maxSpeed, limelight, drive),
+            new AimAndDriveBackCommand(distance, maxSpeed, limelight, drive, lights),
+            new AutonomousShooterLightsCommand(lights),
```

Comment:
Instead of creating a new command here to change the colors to green, can you change the lights to green in the `ShooterCommand` initialization? Just like you did in the `AimAndDriveBackCommand`